sdk/storage/storage-file-datalake/CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # Release History
 
+## 12.20.0 (Unreleased)
+
+### Features Added
+
+### Breaking Changes
+
+### Bugs Fixed
+
+### Other Changes
+
+- Migrated dependency on `@azure/core-http` to `@azure/core-rest-pipeline`.
+
 ## 12.12.0 (Unreleased)
```

Comment:
merge into 12.20.0?

Reply:
I wasn't sure if this was going to ship out of the stable branch or not, definitely have to fix up the changelog before we release out of main